src/gcp/cloudfunctions.js

```diff
@@ -94,6 +94,37 @@ function _createFunction(options) {
     );
 }
 
+/**
+ * Sets the IAM policy of a Google Cloud Function.
+ * @param {*} options Options object.
+ * @param {string} options.projectId Project that owns the Function.
+ * @param {string} options.region Region in which the Function exists.
+ * @param {string} options.functionName Name of the Function.
+ * @param {*} options.policy The [policy](https://cloud.google.com/functions/docs/reference/rest/v1/projects.locations.functions/setIamPolicy) to set.
+ */
+async function _setIamPolicy(options) {
```

Comment:
We have `setIamPolicy` for Extensions elsewhere in the codebase. I'm not super duper concerned, but is it worth considering a shared codebase?

Reply:
I don't think so. These `setIamPolicy` calls are resource-specific - if we are going to consolidate them somehow, it would have to be pretty well thought out and abstracted in order not to break things. I'm not worried about it for now.

Reply:
Agreed that it's better to be separate. The origins are different for the setIamPolicy calls.